src/julia_gc.c

```diff
@@ -826,16 +832,26 @@ void InitBags(UInt initial_size, Bag * stack_bottom, UInt stack_align)
     jl_gc_set_cb_post_gc(PostGCHook, 1);
     // jl_gc_enable(0); /// DEBUGGING
 
+    // Get GapObj abstract julia type
+    jl_module_t * gaptypes_module =
+        (jl_module_t *)jl_get_global(jl_main_module, jl_symbol("GAPTypes"));
+    jl_datatype_t * gapobj_type =
+        (jl_datatype_t *)jl_get_global(gaptypes_module, jl_symbol("GapObj"));
```

Comment:
For both of those, I'd recommend that we test what happens if either symbol is unbound -- do they throw exceptions? Or just return a null pointer? In the latter case, we should handle that.

Also, perhaps we can at least add checks that the values we go there actually represent a module resp. a type object?

Reply:
Yep. I will add the error checks.